source/developers-guide/digital-publishing-elements/index.md

```diff
@@ -22,7 +22,7 @@ Digital Publishing is introduced in Shopware 5.1.0 as a new advanced feature. Th
 
 <img src="img/file_structure.jpg" class="is-float-right" />
 
-We will not describe the hole plugin development in this article, but only the important parts to get started with the custom element. To get more details about developing plugins for Shopware, see the <a href="{{ site.url }}/developers-guide/plugin-quick-start/" target="_blank">quick start guide</a>. In our example we create a plugin called `SwagDigitalPublishingSample`. This is just an example name for our plugin and will be used for some directory and file names. When you create your own plugin, just replace the corresponding parts with the name of your own plugin. 
+We will not describe the whole plugin development in this article, but only the important parts to get started with the custom element. To get more details about developing plugins for Shopware, see the <a href="{{ site.url }}/developers-guide/plugin-quick-start/" target="_blank">quick start guide</a>. In our example we create a plugin called `SwagDigitalPublishingSample`. This is just an example name for our plugin and will be used for some directory and file names. When you create your own plugin, just replace the corresponding parts with the name of your own plugin. 
```

Comment:
The sentence has a negative attribute - you're telling people what isn't covered by the article. It would be better to tell the people out there what are you talking about in this article.

---
In this article we'll be focusing on the important parts to get started with the development of custom digital publishing elements. If you want to get more information about developing plugins for Shopware in general, please head over to the <a href="{{ site.url }}/developers-guide/plugin-quick-start/" target="_blank">plugin quick start guide</a>.